loom-cli/main/loom.hs

```diff
@@ -250,6 +280,53 @@ siteFilterP =
     <> OA.metavar "SITE_FILTER"
     <> OA.help "A filter for generating a sub-set of the site to reduce the time taken"
 
+pathsP :: Parser PathsCommand
+pathsP =
+  PathsCommand
+    <$> OA.argument
+          (OA.eitherTextReader id parseBundle)
+          (OA.metavar "BUNDLE")
+    <*> pathSetP
+    <*> pathGroupingP
+
+parseBundle :: Text -> Either Text HardcodedBundle
+parseBundle = \case
+  "purs" ->
+    pure Purs
+  "purs.test" ->
+    pure PursTest
+  _ ->
+    Left "Unsupported bundle; only available for 'purs' and 'purs.test'"
```

Comment:
FWIW you can just use subparser and command' if you want the help for free

Reply:
I'm not entirely sure how to structure this without going another level of nesting; might have a chat when you're free, if you don't mind.